powerpacks.go

```diff
@@ -123,6 +126,40 @@ type NumberRemoveParams struct {
 	Unrent bool `json:"unrent,omitempty"`
 }
 
+type AddNumberOptions struct {
+	//Service can be 'sms' or 'mms'
```

Comment:
Wondering if it'll be better if these are explicitly defined constants:

In `messages.go`:

```go
const (
    SMS = "sms"
    MMS = "mms"
)
```